src/extensions/http_api/definitions.js

```diff
@@ -135,6 +135,19 @@ exports.install = ({ baseUrl = '' } = {}) => {
         this.state.set(key, _.get(data, path))
     })
 
+    /**
+     * Pick a value from previous json response or header and set it to state
+     */
+    Given(
+        /^(?:I )?replace(?: placeholder)? (.+) in (.+) to ([^\s]+)(?: with regex options? (.+)?)?$/,
+        function (search, key, replaceValue, option) {
```

Comment:
You should test this function. You only tested the definition's regexp in unit tests : no `def.exec`.

Reply:
Yes, tested!